src/posts/paper/99-changelog.ts

```diff
@@ -7,6 +7,12 @@ interface Change {
 }
 
 const changes: Change[] = [
+  {
+    date: new Date(2021, 1, 21),
+    content: `
+* Added precaution for being vaccinated. See paper [Q&A](/paper/13-1-and-a#vaccines-qa) and [Research Sources](/paper/14-research-sources#vaccines) for details.
```

Comment:
This should probably exist on the front page too. "Added 'your vaccine' precaution" for brevity?

Reply:
Ah, I see someone removed the code that puts the changelog on the front page. Grumbles. Done.